windows-gaming-driver/src/driver/qemu.rs

```diff
@@ -269,8 +269,12 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path, clientpipe_path: &Path, monito
         }
     }
 
+    if let Some(ref cmd) = cfg.additional_qemu_cmdline {
+        qemu.args(cmd.split(' '));
+    }
+
     qemu.stdin(Stdio::null());
-    trace!("qemu: {:?}", qemu);
+    debug!("qemu: {:?}", qemu);
```

Comment:
You might wanna change this back to trace! and instead insert a debug!("additional qemu parameters: {}", cmd); inside the if to make it consistent with the rest of the startup.